release/_publishing.sh

```diff
@@ -122,6 +107,39 @@ function check_url {
 	fi
 }
 
+function get_patcher_product_version_label {
+	if [[ "${_ARTIFACT_VERSION}" == 7.3.* ]]
+	then
+		echo "DXP 7.3"
+	elif [[ "${_ARTIFACT_VERSION}" == 7.4.* ]]
```

Comment:
@davisaints, shouldn't we keep `_PRODUCT_VERSION` here? Why did you decide to change?

In theory, we should use `_PRODUCT_VERSION` and only when necessary use `_ARTIFACT_VERSION`. Can you please apply this idea for the other functions too?

Thanks